spec/controllers/idv/review_controller_spec.rb

```diff
@@ -19,6 +19,7 @@
       service_provider: nil,
     )
     idv_session.profile_confirmation = true
+    idv_session.resolution_successful = 'phone'
```

Comment:
silly question... does GPO flow not use this controller?

Reply:
So, long story here. The GPO flow does use this controller. This value, however, is not related at all to the GPO flow or the phone flow for that matter. It is a value that the Flow State Machine set starting a while back that, as best as I can tell, was a copy/paste error.

We've kept using the `phone` value for continuity. I've been talking with @soniaconnolly about removing `resolution_successful` in favor of just using `profile_confirmation`. That may take some work to make sure things don't break in the 50/50 state.

Reply:
For reference, [here](https://github.com/18F/identity-idp/blob/main/app/controllers/concerns/idv/verify_info_concern.rb#L220) is where this gets set when the profile is verified. This code was copied over from the verify step in the flow state machine. You can also see where the likely typo occurred.